docs/basics/object-spawning.md

```diff
@@ -53,6 +53,43 @@ When you set the destroyWithScene property to `false` it will be treated the sam
 
 [Learn more about Netcode Scene Management here](scenemanagement/scene-management-overview.md)
 
+:::caution You might find it useful to add a `GameObject` property in a `NetworkBehaviour` derived component that you use to assign a network prefab instance for dynamically spawning. You need to make sure to instantiate a new instance **prior** to spawning. If you attempt to just spawn the actual network prefab instance it can result in unexpected results.
+:::
+
+### Taking Prefab Overrides Into Consideration
+Sometimes you might want to make a simpler prefab instance to be spawned on server version the override for clients. You should take this into consideration when dynamically spawning a network prefab. If you are running as a host, you want the override to spawn since a host is both a server and a client. However, if you also want to have the ability to run as a dedicated server you might want to spawn the source network prefab. 
+
+**_There are two ways you can accomplish this:_**
+
```

Comment:
```suggestion
Sometimes, you might want to make a simpler prefab instance to be spawned on server version the override for clients. You should take this into consideration when dynamically spawning a network prefab. If you're running as a host, you want the override to spawn since a host is both a server and a client. However, if you also want to have the ability to run as a dedicated server, you might want to spawn the source network prefab. 

There are two ways you can accomplish this, as explained below.

```